### ISSUE TITLE
Delete specific choice isn't working on multiple mode by keyboard

### ISSUE THREAD

**Issue by angelfraga** [select2/select2#5395]
Hi, at first thanks for the awesome library.

I m having accessibility problems when I try to delete one specific choice from the choice list via keyboard. Currently delete all of them is necessary to delete just one.

In previous versions, if the user pressed the left arrow key, it was available to jump into/mark a different choice, then press backspace key and remove it from the list without remove all choices.

I m not really sure if a bug or missing accessibility feature, but could be nice recover it because save a lot time in huge lists and improve a lot the keyboard accessibility.

This issue happens at least from 4.x versions, but the 3.5.3 version was working fine using `select2-search-choice-focus` css class. 

Pressing enter key during the selection is also not working to delete one selected choice. It looks like the only way to delete one specific choice is clicking on the x icon or clicking the choice in the available list. 

Then:
- is it a bug ? 🐛 
- is it a missing feature? 🛩 
- is it intentioned removed feature? 😢  Do you plan to recover it ? 😃 👍 ❤️ 

Thanks again.
